FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/FleetProvisioningDemoExample.c

```diff
@@ -646,6 +646,12 @@ int prvFleetProvisioningTask( void * pvParameters )
              * topics. In this demo we use CBOR encoding for the payloads,
              * so we use the CBOR variants of the topics. */
             xStatus = prvSubscribeToCsrResponseTopics();
+
+            if( xStatus == true )
+            {
+                /* Subscribe to the RegisterThing response topics. */
+                xStatus = prvSubscribeToRegisterThingResponseTopics();
+            }
```

Comment:
Per documentation on AWS for the [Device Provisioning MQTT API](https://docs.aws.amazon.com/iot/latest/developerguide/fleet-provision-api.html) subscribe to relevant topics before publishing to them.


> Important
> Before publishing a request message topic, subscribe to the response topics to receive the response. The messages used by this API use MQTT's publish/subscribe protocol to provide a request and response interaction.
> If you do not subscribe to the response topics before you publish a request, you might not receive the results of that request.